_sass/_components/lists.scss

```diff
@@ -35,6 +35,28 @@
 
 // Project page lists
 
-.partner-info h5 {
-    @include u-padding-y(2);
+.partner-info {
+  a {
+    color: color('accent-cool');
+    text-decoration: underline;
+  }
+
+  .usa-sidenav__sublist a {
+    padding-left: 0;
```

Comment:
could we replace this one with `padding: units(.5) o`

Reply:
👍